tutorials/.enableImplicitMTWrapper.py

```diff
@@ -0,0 +1,14 @@
+# this is meant to be used only to run tutorials as tests
+
+import ROOT
+import sys
+
+
+nCores = int(sys.argv[1])
+tutorialName = sys.argv[2]
+
+if "imt" in ROOT.gROOT.GetConfigFeatures():
+    ROOT.EnableImplicitMT(nCores)
+
+tutorialFile = open(tutorialName)
+exec(tutorialFile.read())
```

Comment:
shouldn't we pass `sys.argv[3] and subsequent)` to the exec ?

Reply:
no because there is no tutorial with more than one argument.

Reply:
then we should throw if there is more than 2 arguments, just in case somebody (in a few years) adds a tutorials that needs an argument.